upup/pkg/fi/nodeup/config.go

```diff
@@ -54,6 +54,9 @@ type NodeUpConfig struct {
 
 // Image is a docker image we should pre-load
 type Image struct {
+	// Name is the name of the tagged image
+	Name string `json:"name,omitempty"`
```

Comment:
This name is the 'docker name' i.e. the name we pass to `docker run`.

Added this:

```
// Name is the name of the tagged image
	// This is the name we would pass to "docker run", whereas source could be a URL from which
	// we would download an image.
	Name string `json:"name,omitempty"`
```